classes/Product.php

```diff
@@ -75,14 +68,6 @@ class ProductCore extends ObjectModel
     /** @var string|array Short description or array of short description by id_lang */
     public $description_short;
 
-    /**
-     * @deprecated since 1.7.8
-     * @see StockAvailable::$quantity instead
-     *
-     * @var int Quantity available
-     */
-    public $quantity = 0;
```

Comment:
Hi, I think we should not remove things deprecated in 1.7.8

Else we make it very hard for people to upgrade from 1.7.8 to 8.0.0 because there is no transition version